test/test_proxy_tensor.py

```diff
@@ -238,6 +238,19 @@ def f2(x):
 
         traced = make_fx(f2)(torch.randn(3))
         self.assertFalse(is_any_sum(traced))
+        self.assertFalse(is_any_sigmoid(traced))
+        self.assertTrue(is_any_digamma(traced))
+
+        # Verify that the forward function of a graph module produced as a 
+        # side effect of an inner `make_fx` is still traced
+        def f3(x):
```

Comment:
Added a test to show that graph module artifact derived from same inputs as external trace has `forward` function that is still traced when called.